src/app/common/interfaces/createUser.ts

```diff
@@ -0,0 +1,8 @@
+export interface CreateUser {
```

Comment:
la fel ca cealalata, asta nu are ce cauta in common din moment ce e folosita doar in create-user module